### ISSUE TITLE
Select2 not focused when tabbing

### ISSUE THREAD

**Issue by KuenzelIT** [select2/select2#3919]
Hey,

I have a simple text input before the select2 multi-select dropdown. When I focus the text input and press "tab", the select2 box gets focused (border gets black), but the cursor is not inside of the select2 field. Another tab gets me to the next input field.

My code looks like this:
```html
<div>
    <input type="text" class="form-control" />
</div>
<div>
    <select class="myselect2" multiple="multiple"></select>
</div>
```

Anyone knows why its behaving that way?

Thanks!

**Comment by kevin-brown** [comment]
This should be fixed in 4.0.1-rc.1, which was released last night.

**Comment by MarcPinnell** [comment]
I am seeing this behavior still with 4.1.0-rc.0
